pkg/remote/google/repository/cloudresourcemanager.go

```diff
@@ -25,26 +28,40 @@ func NewCloudResourceManagerRepository(service *cloudresourcemanager.Service, co
 }
 
 func (s *cloudResourceManagerRepository) ListProjectsBindings() (map[string]map[string][]string, error) {
-	if cachedResults := s.cache.Get("ListProjectsBindings"); cachedResults != nil {
-		return cachedResults.(map[string]map[string][]string), nil
-	}
 
-	request := new(cloudresourcemanager.GetIamPolicyRequest)
-	policy, err := s.service.Projects.GetIamPolicy(s.config.Project, request).Do()
-	if err != nil {
-		return nil, err
-	}
+	bindingsByProject := make(map[string]map[string][]string)
+	errorsByProject := make(map[string]error)
+	var erorsString string
 
-	bindings := make(map[string][]string)
+	for _, scope := range s.config.Scope {
+		if strings.Contains(scope, "projects/") {
+			project := strings.Split(scope, "projects/")[1]
+			request := new(cloudresourcemanager.GetIamPolicyRequest)
+			policy, err := s.service.Projects.GetIamPolicy(project, request).Do()
+			if err != nil {
+				errorsByProject[project] = err
+				bindingsByProject[project] = nil
+				continue
+			}
 
-	for _, binding := range policy.Bindings {
-		bindings[binding.Role] = binding.Members
-	}
+			bindings := make(map[string][]string)
+			for _, binding := range policy.Bindings {
+				bindings[binding.Role] = binding.Members
+			}
 
-	bindingsByProject := make(map[string]map[string][]string)
-	bindingsByProject[s.config.Project] = bindings
+			bindingsByProject[project] = bindings
+
+			s.cache.Put("ListProjectsBindings", bindingsByProject)
 
-	s.cache.Put("ListProjectsBindings", bindingsByProject)
+		}
+	}
 
-	return bindingsByProject, nil
+	if len(errorsByProject) > 0 {
+		for project, errval := range errorsByProject {
+			erorsString = erorsString + "Project: " + project + " had the following error: " + errval.Error() + "; "
+		}
+		return bindingsByProject, errors.New(erorsString)
+	} else {
+		return bindingsByProject, nil
+	}
```

Comment:
this else case is useless, you can just return at the end

Reply:
Agree, implemented